Automattic/woocommerce-payments
# E2E: fix `merchant-orders-full-refund` missing order amount

Fixes #8330

> [!CAUTION]
> This PR targets `trunk` which is required to fix e2e tests for server and client repos.

#### Changes proposed in this Pull Request

This PR fixes the `merchant-orders-full-refund` e2e test suite by changing the target element used to get the order amount.

Rather than get this amount from the merchant order page which is difficult to parse due to multiple unlabelled `.woocommerce-Price-amount` elements, the amount is instead extracted from the shopper order received page.

#### Testing instructions





* Inspect the results of this PR's GH check `E2E Tests - Pull Request / WC - latest | wcpay - merchant (pull_request)`
* Ensure the test suite `merchant-orders-full-refund` passes ✅

-------------------

- [x] Run `npm run changelog` to add a changelog file. **Not suitable for a PR direct to `trunk`**

**Post merge**
